test/integration/other/migrations.js

```diff
@@ -227,3 +229,78 @@ describe('datbase migrations: removing default project', function() {
     projCount.should.equal(0);
   }));
 });
+
+describe('database migrations: 20220908-01-remove-google-backups', () => {
+  // eslint-disable-next-line space-before-function-paren, func-names
+  beforeEach(function() {
```

Comment:
I wouldn't be surprised if we end up wanting to turn off these rules, at least in testing. For now, I'm just disabling these rules in a similar way as #545 so that this commit passes once it's merged into the `master` branch.

Reply:
I'd be up for disabling everywhere.  I'm not sure `func-names` has much value if there's no transpilation applied to the code.